scripts/delete-old-prs.ts

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env -S deno run --allow-env --allow-net --allow-run --allow-read
+// To format: npx prettier --write .
+import { commit, listFiles } from "npm:@huggingface/hub@0.1.3";
```

Comment:
Is that the main reason for picking deno? No need for `package.json` or `npm`? :)

Reply:
(ok I should have read the description lol)